File: beginner/pagina09/2709/2709.py
def eh_primo(numero):
    if numero <= 1:
        return False
    for i in range(2, numero ** 0.5):
        if numero % i == 0:
            return False

    return True

def contagem(moedas, salto):
    index = len(moedas) - 1
    soma = 0
    while index >= 0:
        soma += moedas[index]
        index -= salto
    return soma

while True:
    try:
        n = int(input())
        moedas = []

        for _ in range(n):
            moeda = int(input())
            moedas.append(moeda)

        salto = int(input())

        resultado = contagem(moedas, salto)

        if eh_primo(resultado):
            print("You're a coastal aircraft. Robbie, a large silver aircraft.")
        else:
            print("Bad boy! I'll hit you.")

    except EOFError:
        break

File: beginner/pagina09/2709/2709.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").trim().split('\n')

function eh_primo(n){
    if(n <= 1) return false
    for(let i = 2; i < Math.sqrt(n); i++){
        if(n % i === 0) return false
    }
    return true
}

function contagem(moedas, salto){
    let index = moedas.length - 1
    let soma = 0
    while(index >= 0){
        soma += moedas[index]
        index -= salto
    }
    return soma
}

while(input.length){
    let n = Number(input.shift())
    let moedas = []

    for(let i = 0; i < n; i++){
        moedas.push(Number(input.shift()))
    }

    let salto = Number(input.shift())
    let resultado = contagem(moedas, salto)

    if(eh_primo(resultado)){
        console.log("You're a coastal aircraft. Robbie, a large silver aircraft.")
    } else {
        console.log("Bad boy! I'll hit you.")
    }
}